src/script/interpreter.cpp

```diff
@@ -1146,6 +1235,8 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
             // Size limits
             if (stack.size() + altstack.size() > MAX_STACK_SIZE)
                 return set_error(serror, SCRIPT_ERR_STACK_SIZE);
+
+            ++opcode_pos;
```

Comment:
If we ever had a `continue`, this would get skipped. Seems like a code footgun.

We could replace `while (pc < pend)` with `for ( ; pc < pend; ++opcode_pos)` perhaps?

Reply:
I think there are bigger footguns already if there ever was a continue (like the stack size check just above), but regardless: good idea, done.